src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -196,11 +196,22 @@ internal XmlLoggingConfiguration([NotNull] string xmlContents, [CanBeNull] strin
         /// <summary>
         /// Parse XML string as NLog configuration
         /// </summary>
-        /// <param name="xml">NLog configuration</param>
+        /// <param name="xml">The XML contents.</param>
```

Comment:
proposal: "NLog configuration as XML string"

(I find The XML contents confusing)

Reply:
same below

Reply:
Fixed